src/Components/ArtworkFilter/ForSaleCheckbox.tsx

```diff
@@ -1,59 +1,30 @@
-import React from "react"
+import React, { Component, HTMLProps } from "react"
 
 import Checkbox from "../Checkbox"
 
 import styled from "styled-components"
 import colors from "../../Assets/Colors"
 import { primary } from "../../Assets/Fonts"
 
-interface Props extends React.HTMLProps<ForSaleCheckbox> {
-  onChange?: any
-}
-
-interface State {
-  isChecked: boolean
-}
-
-export class ForSaleCheckbox extends React.Component<Props, State> {
-  state = {
-    isChecked: false,
-  }
-
-  onClick() {
-    this.setState({
-      isChecked: !this.state.isChecked,
-    })
-    this.props.onChange()
-  }
-
-  render() {
-    const { isChecked } = this.state
-    const { checked } = this.props
-    return (
-      <div className={this.props.className} onClick={() => this.onClick()}>
-        <StyledCheckbox checked={isChecked || checked} />
-        <label>Only for Sale</label>
-      </div>
-    )
-  }
-}
-
-const StyledCheckbox = styled(Checkbox)`
-  margin-right: 15px
-`
-
-const StyledForSaleCheckbox = styled(ForSaleCheckbox)`
-  display: inline-block;
+const CheckboxContainer = styled.div`
```

Comment:
@yuki24 - #superminor - most components have the style definition below the main component for API readability.

Reply:
done.